business_objects/record.py

```diff
@@ -309,19 +392,14 @@ def count_missing_rats_records(project_id: str, attribute_id: str) -> int:
     return result.c
 
 
-def count_missing_tokenized_records(project_id: str) -> int:
+def count_missing_tokenized_records(project_id: str) -> List[Any]:
```

Comment:
why does this now return an array (at least going by the function signature)

Reply:
Changed back to int